content/chainguard/libraries/python/build-configuration.md

```diff
@@ -37,75 +38,112 @@ The following steps allow you to determine the URL and authentication details fo
 1. Select the **Packages** tab.
 1. Select **Push/Pull Packages**.
 1. Choose the **PyPI** format.
-1. Copy the value in the `<url>` tag from the XML snippet with the `<repositories>` entry. For example, `https://dl.cloudsmith.io/basic/exampleorg/chainguard-python/python/` with `exampleorg` replaced with the name of your organization. Note the URL contains both the name of the repository `chainguard-python` as well as `python` as an identifier for the format.
-1. Select your desired authentication method (either *Default* or *API Key*). Copy the provided username and password values for configuration of tools. You can perform this step multiple times if you're using different authentication methods for different tools.
+1. Copy the value in the `<url>` tag from the XML snippet with the
+   `<repositories>` entry. For example,
+   `https://dl.cloudsmith.io/basic/exampleorg/chainguard-python/python/` with
+   `exampleorg` replaced with the name of your organization. The URL contains
+   both the name of the repository `chainguard-python` as well as `python` as an
+   identifier for the format.  Note that for use with build tools you must
+   append `simple` to the URL so that the package index is used successfully -
+   `https://dl.cloudsmith.io/basic/exampleorg/chainguard-python/python//simple/`.
+1. Select your desired authentication method (either *Default* or *API Key*).
+   Copy the provided username and password values for configuration of tools.
+   You can perform this step multiple times if you're using different
+   authentication methods for different tools.
+
+<a id="artifactory"></a>
 
-<a name="artifactory"></a>
 ### JFrog Artifactory
 
 The following steps allow you to determine the identity token and URL for accessing your organization's JFrog Artifactory repository manager.
 
 1. Select **Administration** in the top navigation bar.
 1. Select **Repositories** in the left hand navigation.
 1. Select the **Virtual** tab in the repositories view.
-1. Locate the *chainguard-python** repository row and select the elipsis (**...**) in the last column on the right.
+1. Locate the *chainguard-python** repository row and select the ellipsis (**...**) in the last column on the right.
```

Comment:
I do like the word "ellipsis", but do you think we should call it "three dots" instead? Would that be more accessible?

Reply:
Yeah ... let me change instead of just fixing spelling